### PR TITLE
Fix insanity

### LINKED ISSUE #8115: Ghosts can pull magazines out of guns
dreamseeker_2016-02-05_21-30-00

### LINKED ISSUE #8300: Can still pulse wires with boxing gloves on
You can't open the door hacking menu with gloves on, but if you open the menu and THEN put on the gloves, you can still pulse the wires while enjoying the 50% protection.

### LINKED ISSUE #8369: Ghosts can remove R-UST fuel rods
(WEB REPORT BY: gringonius REMOTE: 158.69.55.2:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 9c34bda489ae38505db06c716116c562c36a47e4 
> 
> General description of the issue
> As titled.
> 
> What you expected to happen
> Nothing at all.
> 
> What actually happened
> Clicking on the Fuel Assembly Port as a ghost popped out the fuel rod, making the R-UST slow down to a crawl

### LINKED ISSUE #11016: ghosts can use the rust
(WEB REPORT BY: pvak REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 
> General description of the issue

ghosts can click on the fuel assembly port removing the fuel and stuff

> What you expected to happen
> not happen
> 
> What actually happened

halp ghosts griffing me

> Steps to reproduce if possible

click on the fuel assembly thing of the rust while a ghost

### PR DESCRIPTION
**Pull request by BarneyGumball**
Fixes #8115
Fixes #8300
Fixes #8369
Fixes #11016

:cl:
- bugfix: Ghosts can no longer pull magazines out of guns or mess with R-UST fuel ports.
- bugfix: You can no longer avoid the dexterity/consciousness checks when cutting or pulsing wires on machinery.

### CODE REVIEW

**Review by PJB3005 — APPROVED** [review]

**Comment by clusterfack** [comment]
Did you consider how this affects adminghosts

File: code/datums/wires/wires.dm [hunk 1]
```diff
@@ -174,6 +177,10 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 	return
 
 /datum/wires/proc/CanUse(var/mob/living/L)
+	if(!L.dexterity_check())
+		return 0
+	if(L.incapacitated() || L.lying)
+		return 0
```
Comment (Intigracy) [inline code/datums/wires/wires.dm:183]: Great work here.

**Comment by BarneyGumball** [comment]
Adminghosts can still eject the fuel rod, just not using the verb.

For guns I just never really thought it would be needed.  I'll override `incapacitated()` for observer mobs to be an adminghost check.

**Comment by clusterfack** [comment]
Im talking about the wires procs specifically

**Comment by BarneyGumball** [comment]
```
/datum/wires/proc/Interact(var/mob/living/user)
    if(!istype(user))
        return 0
```

Adminghosts can't view most wire menus anyway.

Now that you mention it though, that `mob/living` check could be replaced now too, seeing as `CanUse()` has some sanity.

**Pull request merged**